express/src/controllers/clients.ts

```diff
@@ -30,6 +30,7 @@ export const showClient: RequestHandler = async (req, res) => {
     const contacts = client.contacts;
     const identityVerificationEnabled = client.identity_verification_enabled;
     const claims = identityVerificationEnabled && client.claims ? client.claims : [];
+    const idTokenSigningAlgorithm = client.hasOwnProperty("id_token_signing_algorithm") ? client.id_token_signing_algorithm : "";
```

Comment:
Can we swap this hasOwnProperty check for 
```
const idTokenSigningAlgorithm = client.id_token_signing_algorithm ?? ""
```
This just handles null/undefined safety a bit better, just trying to avoid the issue explained here https://github.com/govuk-one-login/onboarding-self-service-experience/pull/788/commits/9a312113084504ad6fac8ecddebaadf44c6fff5a, where hasOwnProperty returns true for null/undefined values

Reply:
addressed